README.MD

```diff
@@ -85,6 +85,9 @@ module.exports['swagger-generator'] = {
             '500': { description: 'Internal server error' }
         }
     },
+    options: {
+      excludeDeprecatedPutBlueprintRoutes: true,
```

Comment:
just curious, any reason adding this to `options` what about just adding it directly to https://github.com/theoomoregbee/sails-hook-swagger-generator/blob/f60e61a8345f9dd9ad5c2c3cb319fb9eda251d35/lib/interfaces.ts#L5

Reply:
I feel it's a better ux to just add more options to `SwaggerGenerator`

Reply:
Apologies, I added it this way and meant to ask you but forgot. I will change it as per your comment.